src/components/search/Searchbar.tsx

```diff
@@ -1,11 +1,85 @@
-import React from 'react';
+import { useState } from 'react';
+import styled from 'styled-components';
+
+import { GlassIcon } from '../../asset/icon';
 
 const Searchbar = () => {
-    return (
+  const [clicked, setClicked] = useState<boolean[]>([true, false, false, false, false, false]);
+  const categorys: string[] = ['프로젝트', '이미지', '프로토타입', '스트림', '인물', '무드보드'];
+  const indexs: boolean[] = [true, false, false, false, false, false];
```

Comment:
```suggestion ts
const indexList : boolean[] = new Array(categoryList.length).fill(false);

useEffect(()=> {
indexList[0] = true;
},[])

```
으로도 가능하긴하다앙 ㅎㅎ 고냥 의견~